sites/docs/src/pages/_ViewConfigEditor.js

```diff
@@ -114,15 +119,52 @@ export default function ViewConfigEditor(props) {
     setUrl(nextUrl);
   }
 
+  function sanitiseURLs(urls) {
+    return urls
+      .replace(/ /g, '')
+      .split(/;/)
+      .filter(url => url.match(/^http/g));
+  }
+
+  async function handleConfigGeneration() {
+    setError(null);
+    setUseExampleURL(false);
+    const sanitisedUrls = sanitiseURLs(datasetUrls);
+    await generateConfigs(sanitisedUrls)
+      .then((configJson) => {
+        setPendingJson(JSON.stringify(configJson, null, 2));
+        setLoadFrom('editor');
+      })
+      .catch((e) => {
+        setError(e.message);
+      });
+  }
+
+  useEffect(() => {
+    if (useExampleURL) {
+      handleConfigGeneration();
+    }
+  }, [useExampleURL]);
```

Comment:
```suggestion
```

It may be better if the user still needs to press the "Generate Config" button after clicking "Try an example" so they get a clear understanding that the config was generated from the URL